src/ast/mod.rs

```diff
@@ -438,6 +461,72 @@ impl fmt::Display for Expr {
                 high
             ),
             Expr::BinaryOp { left, op, right } => write!(f, "{} {} {}", left, op, right),
+            Expr::Like {
```

Comment:
Seems to match https://www.postgresql.org/docs/current/functions-matching.html so 👍